.github/new_pipeline/pull_request_template.md

```diff
@@ -0,0 +1,80 @@
+---
+name: New pipeline
+description: Initiate the peer review process for a new pipeline
+title: "[NEW PIPELINE]: "
+labels: ["pipeline", "user contributed", "peer review needed"]
+---
+
+> [!IMPORTANT]
+> To facilitate the peer review of the new pipeline, do not change the structure of this
+> document. Only the parts in comments should be replaced.
+
```

Comment:
I'd like to see here a self-assessment checklist. Something like:

Before submitting this pull request, I have made sure that:
- [ ] All scripts and pipelines, and all their inputs, are described properly.
- [ ] The examples of all scripts and pipeline can be ran successfully.
- [ ] GitHub validations pass.
- [ ] The license of the scripts and pipelines are explicitly written in the metadata.
- [ ] All references are cited in the script and pipeline metadata. Cite the methodologies, but also check the packages used for information on how they should be cited.

etc.